build.sbt

```diff
@@ -119,6 +119,7 @@ lazy val kernelLaws = crossProject(JSPlatform, JVMPlatform, NativePlatform)
   .settings(testingDependencies)
   .jsSettings(commonJsSettings)
   .jvmSettings(commonJvmSettings)
+  .jvmSettings(libraryDependencies += "org.scala-lang.modules" %% "scala-collection-compat" % "2.11.0" % Test)
```

Comment:
Sorry, we'd like to avoid adding this dependency. What do you need it for? Cats already has some internal compatibility shims if I remember correctly :)

Reply:
Btw, I was always wondering why we don't use `scala-collection-compat` for tests... But since there are already some ad-hoc collection compat code in cats/tests I didn't think about it too much...

Reply:
Moved to `ScalaSpecificVersion`.